src/lib/getEnginesNodeFromRegistry.ts

```diff
@@ -11,7 +12,7 @@ import getPackageManager from './getPackageManager'
  * @param [options={}] Options.
  * @returns Promised {packageName: engines.node} collection
  */
-async function getEnginesNodeFromRegistry(packageMap: Index<VersionSpec>, options: Options) {
+async function getEnginesNodeFromRegistry(packageMap: Index<VersionResult>, options: Options) {
```

Comment:
The function only uses version, so why pass in more information than what is used?

Reply:
done